metaworld/envs/assets/sawyer_xyz/sawyer_reach_wall_v2.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<mujoco>
+
+    <include file="shared_config.xml"></include>
+
+    <worldbody>
+        <include file="sawyer_xyz_base.xml"></include>
+
+        <body name="obj" pos="0 0.6 0.015">
+            <joint name="objjoint" type="free" limited='false' damping="0.0" armature="0" />
+            <inertial pos="0 0 0" mass=".75" diaginertia="8.80012e-04 8.80012e-04 8.80012e-04"/>
+            <geom name="objGeom" type="cylinder" pos="0 0 0" solimp="0.99 0.99 0.01"
+                  size="0.03 0.015" rgba="1 0 0 1" solref="0.01 1"
+                  contype="1" conaffinity="1" friction="1 0.1 0.002" condim="4" material="wood"/>
+        </body>
+
+         <body name="wall" pos="0.1 0.75 0.05">
+          <geom rgba="1 0.3 0.3 1" type="box" contype="1" size="0.1 0.005 0.05" name="wall"
+                conaffinity="1" pos="0 0 0.005" mass="1000"/>
+        </body>
+
+        <site name="goal" pos="-0.1 0.8 0.2" size="0.02"
+              rgba="0.8 0 0 1"/>
+
+    </worldbody>
+
+    <actuator>
+        <position ctrllimited="true" ctrlrange="-1 1" joint="r_close" kp="400"  user="1"/>
```

Comment:
IIRC, the reach-push-pick-place envs had 3 separate goal `<site>` tags in V1, which sort of made sense because the envs inherited from each other. In V2, we split them up, so it made sense for the XML files to just have their particular goal `<site>` instead of all 3

Reply:
Yeah we've done this for all the past v2 environments so its consistent, and the naming convention is now simpler.